tracker/tracker-assist/src/CallWindow.ts

```diff
@@ -107,20 +107,34 @@ export default class CallWindow {
   private adjustIframeSize() {
     const doc = this.iframe.contentDocument
     if (!doc) { return }
-    this.iframe.style.height = doc.body.scrollHeight + 'px'
-    this.iframe.style.width = doc.body.scrollWidth + 'px'
+    this.iframe.style.height = `${doc.body.scrollHeight}px`
+    this.iframe.style.width = `${doc.body.scrollWidth}px`
   }
 
   setCallEndAction(endCall: () => void) {
     this.load.then(() => {
       if (this.endCallBtn) {
         this.endCallBtn.onclick = endCall
       }
-    })
+    }).catch(e => console.error(e))
```

Comment:
just put `.catch` in the load definition, then no need to catch it everywhere where it can not be thrown

Reply:
Use `app.debug`. You can pass it as a `private readonly logger=console` object to the constructor since we don't need the full App object here.